apps/filebrowser/src/filebrowser/api.py

```diff
@@ -108,3 +109,22 @@ def touch(request):
   
   request.fs.create(request.fs.join(path, name))
   return HttpResponse(status=200)
+
+@error_handler
+def rename(request):
+  src_path = request.POST.get('src_path')
+  dest_path = request.POST.get('dest_path')
+
+  """If dest_path doesn't have a directory specified, use same dir."""
```

Comment:
Why is this required?

Reply:
Rename api works in both ways :
Ex: renaming user/test to user/test123 
1) You can specify the whole path for the dest_path ie dest_path = user/test123 
2) Usually when the api is called from UI dest_path only consists of the new name for the file or folder with no directory specified ie dest_path = test123. Since a directory isnt specified in this scenario it uses the current directory path when calling request.fs.rename() function.

Reply:
I see, let's add it as a comment before https://github.com/cloudera/hue/pull/3619/files#diff-d8a7a37b125cd146d72b25ec4c75d6c208194ebc09ab376f0445cabc76af1bb1R119?